src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -1,4 +1,26 @@
 package seedu.addressbook.commands;
 
-public class SortCommand {
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+/**
+ * Lists all persons in the address book to the user in sorted order.
+ */
+
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ":\n"
+            + "Displays all persons in the address book as a list with index numbers in a sorted order.\n\t"
+            + "Example: " + COMMAND_WORD;
+
+
+    @Override
+    public CommandResult execute() {
+        List<ReadOnlyPerson> allPersons = addressBook.getAllPersons().immutableListView();
+        return new CommandResult(getMessageForPersonListShownSummary(allPersons), allPersons);
```

Comment:
Perhaps indicate in the commit message that this function simply returns the list without actually returning a sorted list of `allPersons` yet!